File: sistemas/web/ohara/model/Manga.php
<?php

class Manga {
    public $idManga;
    public $titulo;
    public $autor;
    public $sinopse;
    public $capa;
    public $anoDePublicacao;
    public $qtdDeVolumes;
    public $qtdDeCapitulos;
    public $popularidade;
    public $status;

    public function __construct($idManga, $titulo, $status, $sinopse, $anoDePublicacao, $qtdDeVolumes, $qtdDeCapitulos, $popularidade, $capa, $autor) {
        $this->idManga = $idManga;
        $this->titulo = $titulo;
        $this->status = $status;
        $this->sinopse = $sinopse;
        $this->anoDePublicacao = $anoDePublicacao;
        $this->qtdDeCapitulos = $qtdDeCapitulos;
        $this->qtdDeVolumes = $qtdDeVolumes;
        $this->popularidade = $popularidade;
        $this->capa = $capa;
        $this->autor = $autor;
    }
}

?>

File: api/alimentadorOhara/src/main/java/br/com/alimentadorOhara/model/Manga.java
package br.com.alimentadorOhara.model;

import lombok.Data;

@Data
public class Manga {

	private Long idManga;

	private String titulo;

	private String autor;

	private String sinopse;

	private String capa;

	private String anoDePublicacao;

	private int qtdDeVolumes;
	
	private int qtdDeCapitulos;

	private int popularidade;

	private String status;
}
